README.md

```diff
@@ -27,6 +27,19 @@ CREATE TABLE paypal (
   currency TEXT,
   country_code TEXT
 );
+
+CREATE TABLE stripe (
+  id TEXT PRIMARY KEY,
+  timestamp TIMESTAMP WITH TIME ZONE,
+  amount INTEGER,
```

Comment:
The `MONEY` type assumes it's not a 0-decimal currency, so I don't think thats a good idea.

`settle_amount`, just below this line, is a `MONEY` type and reflects the actual amount dontated in USD at the time of the charge.

Keeping this an integer ensures we can convert the zero decimal based currencies in the future if we need to.